.travis.yml

```diff
@@ -7,11 +7,15 @@ node_js:
   - 10.14.2
   - 12
   - 14
+  - 15
   - node
 env:
   - REACT_DIST=latest
   - REACT_DIST=next
   - REACT_DIST=experimental
+  # node15 compat
+  # Revisit once https://github.com/npm/cli/pull/2047 is released.
+  - npm_config_yes=true
```

Comment:
Is there any downside to putting it here rather than adding it only for node 15? That was what I was initially thinking..

Reply:
They said specifically that it is backwards compatible. I do set it conditionally but why bloat the config if it works with every version?

Reply:
It looks [here](https://github.com/npm/cli/issues/1935#issuecomment-706278238) that it's compatible for npm 6 and 7, node 10.2.1 still works with npm 5, but we're running this only for node 10.14.2 so this will pass.. We already wrote there that "technically we support 10.0.0, but some of our tooling doesn't" so we're covered there.
Thanks!